src/linuxX64Main/kotlin/mu/KotlinLoggingConfiguration.kt

```diff
@@ -0,0 +1,8 @@
+package mu
+
+object KotlinLoggingConfiguration {
+    var LOG_LEVEL = KotlinLoggingLevel.INFO
```

Comment:
`object` are automatically frozen by default, so `var`s do not make sense here. You can specify backing fields with `AtomicReference` and use getters and setters here.

Reply:
All properties in `KotlinLoggingConfiguration` now use `AtomicReference`, and have been turned into read only properties.